source/content/secure-runtime-access.md

```diff
@@ -9,7 +9,7 @@ searchboost: 150
 
 Pantheon’s database services use strong random passwords and TLS to encrypt communications by default. Customers seeking additional defense in depth can enable Secure Runtime Access (SRA).
 
-When SRA is enabled, the site's Pantheon MySQL databases will reject logins unless the connection comes through the appropriate [SSH tunnel](/ssh-tunnels/).
+SRA creates an early discard of attempts to connect to persistence instances like MySQL or Redis. When SRA is enabled, the site's Pantheon MySQL databases will reject logins unless the connection comes through the appropriate [SSH tunnel](/ssh-tunnels/).
```

Comment:
s/persistence/persistant?

Reply:
Also, is "creates an early discard" the best phrasing here? What about something like:

> SRA quickly terminates attempts to connect...

Reply:
@alexfornuto I transcribed this part right from @davidstrauss describing how it functions. I wanted to rephrase it similar to your suggestion, but I think the wording here is meant to show that the connection attempts aren't even given a chance.
Maybe?
> SRA actively discards attempts to connect to persistent instances like MySQL or Redis, denying the attempt before it reaches the database.